Chapters/04.02-probability.ipynb

```diff
@@ -541,7 +541,7 @@
     "|:-----------|:------------ |:---------------------------|\n",
     "|Not $A$     |$P(\\neg A)$   |$1-P(A)$                    |\n",
     "|$A$ or $B$  |$P(A \\cup B)$ |$P(A) + P(B) - P(A \\cap B)$ |\n",
-    "|$A$ and $B$ |$P(A \\cap B)$ |$P(A|B) P(B)$               |\n"
+    "|$A$ and $B$ |$P(A \\cap B)$ |$P(A\\|B) \\cdot P(B)$               |\n"
```

Comment:
Bar needs to be escaped otherwise it is read as Markdown not as LaTeX.

Reply:
Ah, thanks!